Microsoft.Recognizers.Text.Number.Tests/TestDecimalAndThousandsSeparators.cs

```diff
@@ -0,0 +1,126 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
+using System.Globalization;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using System.Collections.Immutable;
+using Microsoft.Recognizers.Text;
+using Microsoft.Recognizers.Text.Number;
+
+namespace Microsoft.Recognizers.Text.Number.Tests
+{
+    public class ArabicNumberTestConfiguration : INumberParserConfiguration
+    {
```

Comment:
We need this new configuration to go through all potential combinations of decimal separator and non-decimal separator. EnglishNumberParserConfiguration just uses one combination.